awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -44,9 +39,7 @@ class JobTemplateForm extends Component {
     template: {
       name: '',
       description: '',
-      inventory: '',
```

Comment:
The template type requires numbers for these so this was causing some loud warnings when running tests - I removed the default values here and am passing them in-line to the below to silence the warnings.